director/sort_test.go

```diff
@@ -450,11 +450,20 @@ func TestGetClientLatLong(t *testing.T) {
 		log.SetLevel(log.DebugLevel)
 
 		clientIp := netip.Addr{}
+		assert.False(t, clientIpCache.Has(clientIp))
 		coord := getClientLatLong(clientIp)
 
 		assert.True(t, coord.Lat <= usLatMax && coord.Lat >= usLatMin)
 		assert.True(t, coord.Long <= usLongMax && coord.Long >= usLongMin)
 		assert.Contains(t, logOutput.String(), "Unable to sort servers based on client-server distance. Invalid client IP address")
+		assert.NotContains(t, logOutput.String(), "Retrieving pre-assigned lat/long")
+
+		// Get it again to make sure it's coming from the cache
+		coord = getClientLatLong(clientIp)
+		assert.True(t, coord.Lat <= usLatMax && coord.Lat >= usLatMin)
```

Comment:
Rather than asserting that it's within the same  US bound, assert that the two coordinates are the same.

Reply:
Good call. Done :)